byron/ledger/executable-spec/src/Ledger/Delegation.hs

```diff
@@ -335,13 +271,13 @@ instance STS SDELEG where
         Set.notMember (epochDelegator cert) (st ^. keyEpochDelegations)
 
       epochDelegator :: DCert -> (Epoch, VKeyGenesis)
-      epochDelegator cert = (cert ^. depoch, cert ^. dwho . _1)
+      epochDelegator cert = (cert ^. to depoch, cert ^. to dwho . _1)
```

Comment:
I'd like to discuss the possibility of removing the uses of lenses, since for the STS the do not make much sense (and put the executable spec further apart from the formal spec).